src/components/Animal.js

```diff
@@ -132,6 +160,24 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
 
         // Add if new animal
         } else {
+            if (!type && 
+                    breed.length === 0 && disposition.length === 0 && 
+                    !picture &&!availability && 
+                    !newsItem && !description) {
+                setError(true)
+                return
+            }
+
+            body = {
+                "type": type,
+                "breeds": breed,
+                "disposition": disposition,
+                "pic": picture,
+                "availability": availability,
+                "news_item": newsItem,
+                "description": description
+            }
+
             console.log(body)
```

Comment:
I think if this code it going to the production branch we probably don't need the console.logs

Reply:
Same as above.